lib/transfers/transfer.ex

```diff
@@ -71,9 +71,14 @@ defmodule PINXS.Transfers.Transfer do
   end
 
   def get_line_items(transfer_token, config) do
-    API.get("/transfers/#{transfer_token}/line_items", __MODULE__, config)
+    API.get("/transfers/#{transfer_token}/line_items?per_page=500", __MODULE__, config)
   end
 
+  def get_line_items(transfer_token, page, config) do
+    API.get("/transfers/#{transfer_token}/line_items?page=#{page}&per_page=500", __MODULE__, config)
+  end
```

Comment:
Could this just be `def get_line_items(transfer_token, config, page \\ 1)`?

Reply:
It could, but it would mean changing the position of `config` and would make this different to all the others